docs/user_guide.md

```diff
@@ -10,4 +10,51 @@ Cyclone provides a powerful workflow engine that serves general workflow purpose
 $ make run_examples SCENE=cicd REGISTRIES=<registry>/<project>
 ```
 
-In addition to YAML format manifest files, you can also create your workflows via APIs provided by Cyclone Server. Refer to [API Docs](./swagger-api-docs.md) for detailed APIs.
\ No newline at end of file
+In addition to YAML format manifest files, you can also create your workflows via APIs provided by Cyclone Server. Refer to [API Docs](./swagger-api-docs.md) for detailed APIs.
+
+## Integration
+
+Cyclone provides an integration center for users to integrate external system information, including sensitive information such as `username`, `password`, `token`, etc,
+and Cyclone will store them by Kubernetes `Secret`.
+
+Cyclone supports 5 types of integrations: `SCM`, `Cluster`, `DockerRegistry`, `SonarQube` and `General`, and SCM is subdivided into the following subclasses:
```

Comment:
Cyclone provides 5 builtin type of integrations: `SCM`, `Cluster`, `DockerRegistry`, `SonarQube` and `General`.    Here `SCM` means source code management system like GitHub, GitLab. Cyclone support different types of SCM: `GitHub`, `GitLab`, `BitBucket` and `SVN`.

Reply:
I think these informations can be added into the following tables. What about add a `description` column to illustrate the integration types?
> Here `SCM` means source code management system like GitHub, GitLab. Cyclone support different types of SCM: `GitHub`, `GitLab`, `BitBucket` and `SVN`.

Reply:
@supereagle I think to add to a column to illustrate this is a little redundant since most of the cyclone users know what `GitHub`, `GitLab`, `BitBucket` and `SVN` is.

Reply:
Agree